addon/components/x-form.js

```diff
@@ -71,18 +71,20 @@ export default Ember.Component.extend({
 
   init() {
     this._super(...arguments);
-    this._setUpChangeset();
+    if(!this.get('data')) {
+      throw new Error('x-form needs data');
+    }
```

Comment:
Hmm..... Well, I guess we can punt on this if it's not something that we support. Does x-form currently handle the idea of "newness" of an object, or is that managed separately?